package.json

```diff
@@ -82,7 +82,7 @@
         "@commitlint/cli": "^8.3.5",
         "@commitlint/config-conventional": "^8.3.4",
         "@date-io/moment": "1.3.9",
-        "@flyteorg/flyteidl": "0.21.24",
+        "@flyteorg/flyteidl": "0.23.1",
```

Comment:
Did smoke test, the new field looks good.
Is it expected, that if provided Config is not accessible it is using default value? 
See here:
https://user-images.githubusercontent.com/55718143/158853593-7c554fba-b544-4017-9607-41fe7ddc1c4a.mp4
Otherwise LGTM, and I can merge it later.

Reply:
yes, We should use default value if bucket is not accessible